plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -14,151 +14,25 @@
  * limitations under the License.
  */
 
-import {
-  Entity,
-  RELATION_OWNED_BY,
-  RELATION_PART_OF,
-} from '@backstage/catalog-model';
-import {
-  EntityRefLinks,
-  getEntityRelations,
-} from '@backstage/plugin-catalog-react';
-import { Chip, Grid, makeStyles, Typography } from '@material-ui/core';
+import { Grid } from '@material-ui/core';
 import React from 'react';
-import { AboutField } from './AboutField';
-
-const useStyles = makeStyles({
-  description: {
-    wordBreak: 'break-word',
-  },
-});
+import { defaultFields, fields } from './fields';
 
 /**
  * Props for {@link AboutContent}.
  *
  * @public
  */
 export interface AboutContentProps {
-  entity: Entity;
+  children?: React.ReactNode;
 }
 
 /** @public */
-export function AboutContent(props: AboutContentProps) {
-  const { entity } = props;
-  const classes = useStyles();
-  const isSystem = entity.kind.toLocaleLowerCase('en-US') === 'system';
-  const isResource = entity.kind.toLocaleLowerCase('en-US') === 'resource';
-  const isComponent = entity.kind.toLocaleLowerCase('en-US') === 'component';
-  const isAPI = entity.kind.toLocaleLowerCase('en-US') === 'api';
-  const isTemplate = entity.kind.toLocaleLowerCase('en-US') === 'template';
-  const isLocation = entity.kind.toLocaleLowerCase('en-US') === 'location';
-  const isGroup = entity.kind.toLocaleLowerCase('en-US') === 'group';
+export const AboutContent = (props: AboutContentProps) => {
+  const { children } = props;
+  return <Grid container>{children ? children : defaultFields}</Grid>;
+};
 
-  const partOfSystemRelations = getEntityRelations(entity, RELATION_PART_OF, {
-    kind: 'system',
-  });
-  const partOfComponentRelations = getEntityRelations(
-    entity,
-    RELATION_PART_OF,
-    {
-      kind: 'component',
-    },
-  );
-  const partOfDomainRelations = getEntityRelations(entity, RELATION_PART_OF, {
-    kind: 'domain',
-  });
-  const ownedByRelations = getEntityRelations(entity, RELATION_OWNED_BY);
+AboutContent.fields = fields;
```

Comment:
I didn't find a way to attach this to `EntityAboutCard` since it's exposed via `createComponentExtension`, so I put it in `AboutContent`.

Reply:
Hmm, wondering if we really want to add it to the `EntityAboutCard` or maybe just export them as something else. Not sure it belongs in the `AboutContent` really. Fields a little strange.

Reply:
Yes it's a bit weird indeed. I think I can export them directly from `@backstage/plugin-catalog`, but I'm not sure about the naming. WDYT we name `fields` as `AboutCardBuiltInFields` and `defaultFields` as `AboutCardDefaultFields`? There is an exported `AboutField` component already, which makes the naming here a bit tricky. Do you have any name suggestions for this? @benjdlambert